src/main/scala/org/scaladebugger/samples/watchpoints/SingleWatchpointExample.scala

```diff
@@ -28,14 +28,10 @@ object SingleWatchpointExample extends App {
     val fieldName = "x"
 
     println(s"Watching $otherClassName.$fieldName")
-    s.onUnsafeModificationWatchpoint(otherClassName, fieldName).foreach(e => {
-      // Implicit enables conversion of standard value to wrapper value
```

Comment:
This was only change that I was not sure about.
The type returned by e.currentValue did not appear to confirm to the implicit conversion.